version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.3.5-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.3.6-SNAPSHOT"
```

Comment:
I _think_ we probably need to increment more than just the patch version if we've moved to new major versions of many of the dependencies, but I'm not sure what the best practice is... @velvia/@rjmac?